patterns/design-patterns-behavioral/src/test/java/com/baeldung/nulls/UsingOptionalUnitTest.java

```diff
@@ -39,4 +39,10 @@ public void whenArgIsFalse_thenChainResponseAndThrowException() {
 
         assertThrows(Exception.class, () -> classUnderTest.process(false).orElseThrow(() -> new Exception()));
     }
+
+    @Test()
+    void givenEmptyList_whenProcessMulti_throwIllegalArgument() {
+        assertThrows(IllegalArgumentException.class, () -> classUnderTest.findFirst());
```

Comment:
This may be a big change, but `classUnderTest` is a terrible name for the thing we're testing.

`dataObject` would be better. You should never name an object `class` unless it is an object of type `Class` - i.e. the metadata of a class.